### ISSUE TITLE
'React/RCTView.h' file not found in AIRMapCallout.h

### ISSUE THREAD

**Issue by SupriyaKalghatgi** [lelandrichardson/react-native-maps#457]
```
#import <Foundation/Foundation.h>
#import "React/RCTView.h"
		 ^
```

Replacing `#import "React/RCTView.h"` by `#import "RCTView.h"` always in `node_modules` is very annoying.

**Comment by grabcode** [comment]
Same here :( (thanks btw, and keep up the good work people!).

**Comment by jlietart** [comment]
Same issue here :(.

**Comment by steinhartalex** [comment]
+1. Still having the same issue !

**Comment by RocKhalil** [comment]
Here's an answer for your problem:
[https://github.com/lelandrichardson/react-native-maps/issues/371](url)

**Comment by RocKhalil** [comment]
You can find the answer [here](https://github.com/lelandrichardson/react-native-maps/issues/371)

**Comment by spikebrehm** [comment]
I just released 0.8.0, which fixes this issue.
